app/helpers/layout_helper.rb

```diff
@@ -294,14 +294,28 @@ def alert(opts = {})
     content_tag :div, :class => html_class do
       result = "".html_safe
       result += alert_close if opts[:close]
-      result += alert_header(opts[:header])
-      result += opts[:text].html_safe
+      result += alert_header(opts[:header], opts[:class])
+      result += "#{opts[:text]}".html_safe
```

Comment:
Interpolation `#{string}` performs `.to_s` on the object you call it on, in this case `opts[:text]`